solutions/swb-reference/integration-tests/tests/multiStep/user.test.ts

```diff
@@ -26,83 +27,140 @@ describe('multiStep users integration test', () => {
     await paabHelper.cleanup();
   });
 
-  test('Create read update delete user happy path', async () => {
-    // create user
-    const mockUserInput = {
-      firstName: 'Test',
-      lastName: 'User'
-    };
-
-    const { data } = await adminSession.resources.users.create({
-      ...mockUserInput,
-      email: `success+activate-deactivate-user-${uuidv4()}@simulator.amazonses.com`
-    });
-    const mockUserId = data.id;
-
-    const mockUser = {
-      ...mockUserInput,
-      id: mockUserId
-    };
-
-    const { data: acutalUserGetByITAdmin } = await adminSession.resources.users.user(mockUserId).get();
-    expect(acutalUserGetByITAdmin).toMatchObject(mockUser);
-
-    // test PA should see the same result of  `List Users` as ITAdmin
-    const { data: allUsersListedByITAdmin } = await adminSession.resources.users.get();
-    const { data: allUsersListedByPA } = await pa1Session.resources.users.get();
-    expect(allUsersListedByPA).toEqual(allUsersListedByITAdmin);
-
-    // test pagination
-    await adminSession.resources.users.create({
-      ...mockUserInput,
-      email: `success+activate-deactivate-user-${uuidv4()}@simulator.amazonses.com`
+  describe('multistep', () => {
+    test('Create read update delete user', async () => {
+      // create user
+      const mockUserInput = {
+        firstName: 'Test',
+        lastName: 'User'
+      };
+
+      console.log('ITAdmin creating a user');
+      const { data } = await adminSession.resources.users.create({
+        ...mockUserInput,
+        email: `success+activate-deactivate-user-${uuidv4()}@simulator.amazonses.com`
+      });
+      const mockUserId = data.id;
+
+      const mockUser = {
+        ...mockUserInput,
+        id: mockUserId
+      };
+
+      console.log('ITAdmin getting the user created');
+      const { data: acutalUserGetByITAdmin } = await adminSession.resources.users.user(mockUserId).get();
+      expect(acutalUserGetByITAdmin).toMatchObject(mockUser);
+
+      // test PA should see the same result of  `List Users` as ITAdmin
+      console.log('ITAdmin and ProjectAdmin listing all users');
+      const { data: allUsersListedByITAdmin } = await adminSession.resources.users.get();
+      const { data: allUsersListedByPA } = await pa1Session.resources.users.get();
+      expect(allUsersListedByPA).toEqual(allUsersListedByITAdmin);
```

Comment:
Small risk here if another user test that creates users is running at the same time as this test and the results end up differing. See if the team has any concerns with this risk, otherwise we can keep this as is and see if this becomes a flaky test down the road.